DatadogCore/Sources/Datadog.swift

```diff
@@ -105,6 +105,9 @@ public struct Datadog {
         /// The bundle object that contains the current executable.
         public var bundle: Bundle
 
+        /// Flag that determines if UIKit's background tasks are utilized to perform uploads in background.
+        public var backgroundTasksEnabled: Bool
+
         /// Creates a Datadog SDK Configuration object.
         ///
         /// - Parameters:
```

Comment:
I think we should be very clear in the doc that this property will extend the lifecycle of the app up to 30s in background to proceed with data upload. It should also be added to the `init` list of parameters 👍